compiler-testing/src/main/kotlin/com/squareup/anvil/compiler/testing/CommonNames.kt

```diff
@@ -42,7 +42,7 @@ internal operator fun ClassLoader.get(fqName: String): Class<*> = loadClass(fqNa
 internal operator fun ScanResult.get(fqName: FqName): ClassInfo = getClassInfo(fqName)
 internal operator fun ScanResult.get(fqName: String): ClassInfo = getClassInfo(fqName)
 
-internal val Compile2Result.contributingObject: Class<*>
+public val Compile2Result.contributingObject: Class<*>
```

Comment:
Sorry, I didn't notice this since I only wrote tests in this project.  I'll get the rest of the visibility changes.

Reply:
All good! I think defaulting to limited visibility until it's needed makes sense anyway